backend/app/Controllers/Http/ResetPasswordController.js

```diff
@@ -0,0 +1,26 @@
+const { parseISO, isBefore, subHours } = require('date-fns');
+
+const Token = use('App/Models/Token');
+
+class ResetPasswordController {
+  async store({ request, response }) {
```

Comment:
Por essa controller usar apenas a model de User, a função que ela executa sera que não faria sentido dentro da controller de User?
```suggestion
  /**
   * Change and stores user new password.
   * POST password/reset
   *
   * @param {Request} ctx.request
   * @param {Response} ctx.response
   */
  async store({ request, response }) {
```